inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/experimental_detectron_detection_output.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include "single_layer_tests/experimental_detectron_detection_output.hpp"
+
+using namespace ov::test;
+using namespace ov::test::subgraph;
+
+namespace {
+
+const std::vector<float> score_threshold = { 0.01000000074505806f };
+
+const std::vector<float> nms_threshold = { 0.2f };
+
+//// specifies maximal delta of logarithms for width and height
+const std::vector<float> max_delta_log_wh = { 2.0f };
+
+// specifies number of detected classes
+const std::vector<int64_t> num_classes = { 2 };
+
+// specifies maximal number of detections per class
+const std::vector<int64_t> post_nms_count = { 500 };
+
+// specifies maximual number of detections per image
+const std::vector<size_t> max_detections_per_image = { 5 };
+
+// a flag specifies whether to delete background classes or not
+// `true`  means background classes should be deleted,
+// `false` means background classes shouldn't be deleted.
+const std::vector<bool> class_agnostic_box_regression = { true };
+
+// specifies deltas of weights
+const std::vector<std::vector<float>> deltas_weights = { {10.0f, 10.0f, 5.0f, 5.0f} };
+
+const std::vector<std::vector<InputShape>> inputShapes = {
+        // inputRois / inputDeltas / inputScores / inputImInfos
+        static_shapes_to_test_representation({{16, 4}, {16, 8}, {16, 2}, {1, 3}}),
+        {
+            {{-1, -1}, {{16, 4}}},
```

Comment:
need to infer 2 time at least with same shape I guess

Reply:
done, thanks